index.html

```diff
@@ -90,4 +90,13 @@ <h3 class="post-subtitle">
      </article>
     {% endfor %}
   </div>
+
+  <!-- 2020 fundraiser sponsor logos -->
```

Comment:
You can use `<hr>` here for a horizontal divider.

Reply:
Not a fan of hr but ok.